tests/integration/is-empty-test.js

```diff
@@ -109,6 +109,34 @@ module('Integration | Macro | isEmpty', function() {
     });
   });
 
+
```

Comment:
I don't think this is necessary. We don't need to test Ember's implementation of isEmpty, we can trust that it works.